src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy

```diff
@@ -332,4 +355,62 @@ class SwitchHelper {
             assert Math.abs(expected - actual) <= 1
         }
     }
+
+    /**
+     * Disconnect a switch from controller either removing controller settings inside an OVS switch
+     * or blocking access to floodlight via iptables for a hardware switch.
+     *
+     * @param sw - switch which is going to be disconnected.
+     * @param waitForRelatedLinks - it is an optional parameter. Please, use this parameter if you want to be sure
```

Comment:
I feel a bit uncomfortable with 'please' and 'you' words in docs. Make it more neutral and laconic

Reply:
fixed